configuration/analyzer_config.json

```diff
@@ -816,6 +816,29 @@
     "description": "Query a domain or URL against URLhaus API",
     "python_module": "urlhaus_run"
   },
+  "UrlScan_Submit_Result": {
+    "type": "observable",
+    "observable_supported": ["url"],
+    "external_service": true,
+    "description": "Submit & retrieve result of a URL against URLScan API",
+    "requires_configuration": true,
+    "python_module": "urlscan_run",
+    "additional_config_params": {
+      "api_key_name": "URLSCAN_API_KEY",
+      "urlscan_analysis": "submit_result"
+    }
+  },
+  "UrlScan_Search": {
+    "type": "observable",
+    "observable_supported": ["ip", "domain", "url", "hash"],
+    "external_service": true,
+    "description": "Search an IP/domain/url/hash against URLScan API",
+    "requires_configuration": true,
+    "python_module": "urlscan_run",
+    "additional_config_params": {
+      "urlscan_analysis": "search",
```

Comment:
since you declared 2 different configurations, one for submit and other for search, so there's no need for passing an additiional config parameter.